debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerSnapshotChangeEventSource.java

```diff
@@ -218,10 +221,45 @@ protected void complete(SnapshotContext snapshotContext) {
      * @return a valid query string
      */
     @Override
-    protected Optional<String> getSnapshotSelect(SnapshotContext snapshotContext, TableId tableId) {
+    protected Optional<String> getSnapshotSelect(RelationalSnapshotContext snapshotContext, TableId tableId) {
+        String modifiedColumns = checkExcludedColumns(tableId);
```

Comment:
Ok, I think it's a bit unfortunate that this was done _only_ for SQL Server. I've logged https://issues.redhat.com/browse/DBZ-2525 for pulling this up so that all relational connectors benefit from this.